.github/workflows/cloudflare-publish.yml

```diff
@@ -52,23 +75,21 @@ jobs:
         run: pnpm nx run-many --target=build
         env:
           # NOTE: @dxos/cli is the only thing consuming these at build time, the apps all consume them at bundle.
+          DX_SENTRY_DESTINATION: ${{ env.CLI_SENTRY_DESTINATION }}
+          DX_TELEMETRY_API_KEY: ${{ env.CLI_SEGMENT_API_KEY }}
           NODE_ENV: production
-          DX_ENVIRONMENT: staging
-          DX_IPDATA_API_KEY: ${{ secrets.IPDATA_API_KEY }}
-          DX_SENTRY_DESTINATION: ${{ secrets.CLI_SENTRY_DESTINATION }}
-          DX_TELEMETRY_API_KEY: ${{ secrets.CLI_SEGMENT_API_KEY }}
 
-      - name: Bundle Shell
-        run: pnpm nx bundle shell
+      - name: Bundle Apps
+        run: pnpm nx run-many --target=bundle --parallel=3
         env:
           NODE_ENV: production
-          DX_ENVIRONMENT: production
+          NODE_OPTIONS: "--max_old_space_size=12288"
```

Comment:
does it actually need this much? maybe we shouldn't parallel 3 😅

Reply:
I had just grabbed what I knew to work out of the Dockerfile. Is there any downside other than masking the issue of bundle getting too large?

What is the benefit of parallelization?

Reply:
yeah I think the main downside is masking the growing memory usage

main benefit of parallelization is that it runs faster